docs/paper/dev/getting-started/plugin-yml.md

```diff
@@ -1,5 +1,5 @@
 ---
-slug: /dev/plugin-yml
+slug: /dev/getting-started/plugin-yml
```

Comment:
I'd drop this, to match the Velocity docs. If this is added though, also add a redirect in `vercel.json`